.circleci/config.yml

```diff
@@ -52,18 +52,6 @@ jobs:
           key: v9-cache-unix-{{ checksum "package-lock.json" }}
```

Comment:
There's also the `orbs: win`, `restore_dependency_cache_win`, and `win_box` steps that can be removed